README.md

```diff
@@ -52,6 +52,7 @@ draft            | This issue needs more work
 duplicate        | This issue or pull request already exists
 enhancement      | New feature or request
 fluff            | Minor app improvement / quality of life change
+i18n             | Issues regarding translations/localisation support
```

Comment:
```suggestion
i18n             | Issues regarding translations/localization support
```

Reply:
This will make the Queen very unhappy.

Reply:
why

Reply:
the Queen :trollface:

Reply:
> This will make the Queen very unhappy.
> 
> (on a serious note: everything is kept to British English)

Now we can do it

Reply:
This will make the King very unhappy :trollface: